### PR TITLE
Remove httpuv_decodeURIComponent workaround

### PR DESCRIPTION
**Pull request by nealrichardson**
Just noticed this "we can probably remove this in a year" comment... from 2017.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# httpuv 1.6.11.9000
+
+* Remove a workaround to support `shiny` older than version 1.0.6 (2017)
```
Comment (wch) [inline NEWS.md:3]:
```suggestion
* Remove a workaround to support `shiny` older than version 1.0.6 (#378)
```

**Review by wch — APPROVED** [review]
Looks good, thanks!

File: NEWS.md (hunk 1 above)
Comment (nealrichardson) [reply NEWS.md]: FTR I meant this as a year, not PR number, but this is fine

Comment (wch) [reply NEWS.md]: Ah, that makes sense.

**Pull request merged**